scripts/lib-franklin.js

```diff
@@ -541,6 +541,7 @@ export function decorateButtons(element, options = {}) {
             a.appendChild(span);
           }
           if (a.querySelector('span.icon')) {
+            a.setAttribute('aria-label', (options.areaLabel || a.title) || 'Learn more about');
```

Comment:
If a link has a `title` it will be used in lieu of `aria-label`. Does Google LHS require both a `title` and `aria-label` on links?